src/SolverInterface/DspOsiCpx.h

```diff
@@ -13,18 +13,26 @@
 #include "cplex.h"
 #include "OsiCpxSolverInterface.hpp"
 #include "SolverInterface/DspOsi.h"
+#include "Utility/DspMessage.h"
 
 class DspOsiCpx : public DspOsi {
 public:
 
 	/** default constructor */
-	DspOsiCpx() {
+	DspOsiCpx() : 
+	ismip_(false),
+	isqp_(false),
+	isqcp_(false) 
```

Comment:
These may better be defined in `DspOsi` so that other solvers can use them.

Reply:
Abstractized QC related functions and variables in DspOsi